src/needs-refactoring/components/CombinedFeaturePanel/components/AccessibilitySection/OpeningHoursValue.tsx

```diff
@@ -154,11 +153,11 @@ export default function OpeningHoursValue(props: {
   const shownElements = intersperse(shownValue.split(/;|\|\|/), <br />);
 
   if (!outputs.length) {
-    return <>{shownElements}</>;
+    return <div style={{ padding: "var(--space-1)" }}>{shownElements}</div>;
```

Comment:
suggestion (non-blocking):

Please try to prevent Inline-styles whenever possible. Especially, when there is a alternative solution. 🙂 

```suggestion
    return <Box p="1'>{shownElements}</Box>;
```